roles.md

```diff
@@ -4,16 +4,7 @@
 
 The *Cultivator* is responsible for tending to the all hands meetings and important shared fields of the co-operative: where we make decisions (Loomio), where we keep our documents (GitHub), and where we keep our calendar (Google Calendar).
 
-**Tasks**:
--  chairing all hands meetings. This includes...
-   - preparing a pad in advance so we start on time
-   - asking for agenda items (start of meeting)
-   - facilitating discussion
-   - keeping time
-- adding reminders to calendar after meeting
-- reviewing and merging notes
-- as needed, set up loomio theads and decisions
-- as needed, sending reminders of action items
+**Tasks** are described in Guide for [hosting a meeting](#hosting-a-meeting).
```

Comment:
Is this migration an ok thing? I did it when I realized the cultivator role had some things that perhaps could belong in a general guide

Reply:
```suggestion
**Tasks** are described in Guide for [hosting a meeting](/guides.md#hosting-a-meeting).
```

Reply:
This just seems like more clicking around honestly. I would rather read from my _ROLE_ as opposed to _TASK_ if I'm trying to quickly get up to speed (this comes from my experience working from many a training manual and incomplete job handovers)

Reply:
Thanks for feedback! But.... I know less clicking is nice, but these responsibilities aren't just for the all-hands cultivator, right?

Can you express more clearly the sort of thing you're hoping we could do to accommodate your concern? I'm interpreting it as "let's keep slightly different versions in both places", which is unexpected :)

Reply:
I'm unsure I understand why my proposal would require keeping info in two places, can we have the "role" be the page where everything lives?

Reply:
But the cultivator isn't the only role that will ever host a meeting. Someone who's not cultivator might need to schedule a meeting, and doesn't it make more sense to point them to "this is where we doc how we schedule meetings", instead of saying "looking at the things the cultivator does for meetings"

Maybe I'm misunderstanding the scope of what a cultivator is though. Is "cultivator" a term for a role in any meeting, or is it a term for a role in the organization / working group?